content/en/case-studies/northwestern-mutual/index.html

```diff
@@ -22,7 +22,7 @@ <h1> CASE STUDY:<img src="/images/northwestern_logo.png" style="margin-bottom:-1
 <div class="cols">
   <div class="col1">
     <h2>Challenge</h2>
-    In the spring of 2015, Northwestern Mutual acquired a fintech startup, LearnVest, and decided to take "Northwestern Mutual’s leading products and services and meld it with LearnVest’s digital experience and innovative financial planning platform," says Brad Williams, Director of Engineering for Client Experience, Northwestern Mutual. The company’s existing infrastructure had been optimized for batch workflows hosted on on-prem networks; deployments were very traditional, focused on following a process instead of providing deployment agility. "We had to build a platform that was elastically scalable, but also much more responsive, so we could quickly get data to the client website so our end-customers have the experience they expect," says Williams. 
+    In the spring of 2015, Northwestern Mutual acquired a fintech startup, LearnVest, and decided to take "Northwestern Mutual’s leading products and services and meld it with LearnVest’s digital experience and innovative financial planning platform," says Brad Williams, Director of Engineering for Client Experience, Northwestern Mutual. The company’s existing infrastructure had been optimized for batch workflows hosted on on-prem networks; deployments were very traditional, focused on following a process instead of providing deployment agility. "We had to build a platform that was elastically scalable, but also much more responsive, so we could quickly get data to the client website so our end-customers have the experience they expect," says Williams.
```

Comment:
hello @furukawa3 . I'll try and review each file.
Generally, an across-the-file-system merge requires a lot of back merging for all.
Also, the rendering of each changed file should be checked.
I wonder if these changes should wait until after the 1.17 release.
Line 25: Is there a newline or space removed? I don't see a change.

Reply:
Hi @kbhawkey ,  I see.  it's OK to wait until 1.17.  I'll rebase this patch after that.

> Line 25: Is there a newline or space removed? I don't see a change.

I removed a trailing space.  You can check it by copy&paste into other editor (e.g. Vim)  :)